mailserver/mailserver_test.go

```diff
@@ -20,22 +20,25 @@ import (
 	"bytes"
 	"crypto/ecdsa"
 	"encoding/binary"
+	"errors"
+	"fmt"
 	"io/ioutil"
 	"testing"
 	"time"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/rlp"
 	whisper "github.com/ethereum/go-ethereum/whisper/whisperv6"
 	"github.com/status-im/status-go/geth/params"
+	"github.com/stretchr/testify/suite"
 )
 
 const powRequirement = 0.00001
-const peerID = "peerID"
 
 var keyID string
-var shh *whisper.Whisper
 var seed = time.Now().Unix()
+var shh *whisper.Whisper
```

Comment:
Why this `shh` has to be global and cannot be part of the suite with a different name?